sympy/utilities/autowrap.py

```diff
@@ -214,10 +214,12 @@ class CythonCodeWrapper(CodeWrapper):
             "from distutils.core import setup\n"
             "from distutils.extension import Extension\n"
             "from Cython.Distutils import build_ext\n"
+            "import numpy as np\n"
             "\n"
             "setup(\n"
             "    cmdclass = {{'build_ext': build_ext}},\n"
-            "    ext_modules = [Extension({ext_args}, extra_compile_args=['-std=c99'])]\n"
+            "    ext_modules = [Extension({ext_args}, extra_compile_args=['-std=c99'])],\n"
+            "    include_dirs = [np.get_include()],\n"
```

Comment:
Maybe the `numpy` lines of the template should also depend on `_need_numpy`? Assuming that that bit of code works the way I intended, this would allow scalar expressions to be compiled without NumPy being installed.